NameToAscii/NameToAscii.py

```diff
@@ -0,0 +1,129 @@
+me = input("Enter your name: \n\n")
```

Comment:
all the fixes as requested in the previous pr, remove all the above three files, just keep this file. Also move this file inside the conversion directory.